pkg/apis/componentconfig/types.go

```diff
@@ -712,3 +712,15 @@ type PersistentVolumeRecyclerConfiguration struct {
 	// in a multi-node cluster.
 	IncrementTimeoutHostPath int32 `json:"incrementTimeoutHostPath"`
 }
+
+const (
+	ApiServerConfigMapName string = "apiserver-config"
+	ApiServerConfigMapKey  string = "configuration"
+)
+
+// Configuration of the API server. Kept in ApiServerConfigMapName config map under ApiServerConfigMapKey key.
+type ApiServerConfiguration struct {
+	unversioned.TypeMeta
+
+	NumberOfMasterReplicas int32 `json:"numberOfMasterReplicas"`
```

Comment:
how does this map to `cmd/kube-apiserver/app/options/options.go` - other components are making these things converge, right?